### DISCUSSION TITLE
pep8speaks suspended

### DISCUSSION THREAD

**Discussion by effigies** [nipreps/GOVERNANCE#43] [Development]
Does anybody still want `pep8speaks` on their repository? I don't universally use flake8 anymore, so I don't configure its settings for new repositories. I would like to disable it for some repositories, but that means I need to set it to a whitelist, and I don't know who's using it.

**Comment by oesteban** [comment]
I believe we can move away from `pep8speaks` as a general recommendation. Do you want to bring this up in the next monitoring meeting (next week, I believe, cc/ @esavary).
